core/admin/templates/admin_details_wrapper.template.php

```diff
@@ -42,4 +41,4 @@
     </div>
     <!-- post-body -->
 </div>
-<!-- <?php echo $admin_page_wrapper_div_id; // already escaped ?> -->
\ No newline at end of file
+<!-- <?php echo $admin_page_wrapper_div_id; ?> -->
```

Comment:
ooo can you plz add a blank line at the end of this file to satisfy the PHP linter?